aroquev/src/main/webapp/script.js

```diff
@@ -40,3 +40,36 @@ function calculateArmandoAge() {
     ageContainer.innerText = Math.abs(ageDt.getUTCFullYear() - 1970);
     
 }
+
+// Scripts for the image gallery
+var slideIndex = 1;
+
+function initSlides() {
+  showSlides(slideIndex);
+}
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+// Thumbnail image controls
+function currentSlide(n) {
+  showSlides(slideIndex = n);
+}
+
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  var dots = document.getElementsByClassName("dot");
+  if (n > slides.length) {slideIndex = 1}
+  if (n < 1) {slideIndex = slides.length}
+  for (i = 0; i < slides.length; i++) {
```

Comment:
Prefer "for-of" instead of traditional "for" when iterating over arrays.

```
for (const slide of slides) {
  // do something with slide
}
```

https://google.github.io/styleguide/jsguide.html#features-for-loops

Reply:
Done